### PR TITLE
Completed Leng homework

### PR DESCRIPTION
**Pull request by techCircle**
- there are 2 homeworks

### CODE REVIEW

**Review by techCircle — COMMENTED** [review]
test1

File: homework-from-leng [hunk 1]
```diff
@@ -2,3 +2,7 @@ leng homework
 and homework again 
 .end
```
Comment (techCircle) [inline homework-from-leng:3]: I will fix this line later in the next pr

**Review by Almire24 — CHANGES_REQUESTED** [review]
test1

File: homework-from-leng [hunk 2]
```diff
@@ -2,3 +2,7 @@ leng homework
...
 q
+
+
+Homework is done. It ready to submit.
```
Comment (techCircle) [inline homework-from-leng:7]: What does these mean? Could you explain more?

**Review by Almire24 — APPROVED** [review]

**Review by LengSainoy — CHANGES_REQUESTED** [review]
redo this!!

**Review by Almire24 — CHANGES_REQUESTED** [review]
changes

**Review by Almire24 — APPROVED** [review]

**Review by mrshihi — APPROVED** [review]

**Review by Almire24 — CHANGES_REQUESTED** [review]
good to go

**Review by LengSainoy — APPROVED** [review]
LGTM!!

**Review by Almire24 — APPROVED** [review]
good job

**Review by Flora1205 — APPROVED** [review]

**Pull request merged**